lib/translator/inquiry.cc

```diff
@@ -187,6 +187,56 @@ void TranslateUnitSerialNumberVpd(
   memcpy(&buffer[sizeof(result)], &product_serial_number, result.page_length);
 }
 
+void TranslateBuildBlockLimitsVpd(
```

Comment:
I'm not too sure how the inquiry command works but will this function be part of the interface called by the engine or is it mainly a helper function? If it is part of the interface, can we add a `ToScsi` suffix to the name?

Of if you're against the `ToScsi`, I feel like having both `Translate` and `Build` in the name is confusing so maybe just have one